proto/README.md

```diff
@@ -14,8 +14,11 @@ custom | - | [CustomMessage](#custommessage) | SENSR additional result message (
 Field | Unit | Type | Description
 --- | --- | --- | ---
 objects | - | [Object](#object) | object list in the latest frame.
+has_objects | - | bool | A flag to check `objects` field is valid or not
 zones | - | [ZoneConfig](https://github.com/seoulrobotics/sensr_proto/blob/master/output.proto) | general zone configuration. (Freq. appx. 10s) 
+has_zones | - | bool | A flag to check `zones` field is valid or not
 health | - | [SystemHealth](https://github.com/seoulrobotics/sensr_proto/blob/master/output.proto) | system health in the latest frame. (Freq. appx. 10s) 
+static_object | - | [Object](#object) | static object list in the latest frame. 
```

Comment:
Sometimes we are starting uppercase and somtimes not:

object list in the latest frame.

vs.

A flag to check `zones` field is valid or not